node-launchpad/src/components/manage_nodes.rs

```diff
@@ -82,13 +90,9 @@ impl Component for ManageNodes {
         let send_back = match key.code {
             KeyCode::Enter => {
                 let nodes_to_start_str = self.nodes_to_start_input.value().to_string();
-                let nodes_to_start = {
-                    let max_space_to_use = std::cmp::min(
-                        self.get_nodes_to_start() * GB_PER_NODE,
-                        self.available_disk_space_gb,
-                    );
-                    max_space_to_use / GB_PER_NODE
-                };
+                let nodes_to_start =
+                    std::cmp::min(self.get_nodes_to_start_val(), self.max_nodes_to_start());
```

Comment:
Maybe this should be also in `max_nodes_to_start`? Passing `get_nodes_to_start_val()` as a param?